libs/design-system/snackbar/src/lib/snackbar.component.stories.ts

```diff
@@ -0,0 +1,137 @@
+import { Component, importProvidersFrom, inject, input } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { applicationConfig, Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+import { SnackbarService } from './snackbar.service';
+import { provideDesignSystem } from '@hra-ui/design-system';
+import { ButtonPosition, SnackbarData } from './snackbar.component';
+
+@Component({
+  selector: 'hra-snackbar-demo',
+  standalone: true,
+  imports: [MatButtonModule],
+  template: `
+    <button mat-flat-button (click)="snackbar.open(message(), action(), close(), actionButtonPosition())">
```

Comment:
Pass duration as the 5th argument in the config object. https://material.angular.io/components/snack-bar/api#MatSnackBarConfig

Reply:
Maybe move the logic into a simple method to not bloat the template